go.mod

```diff
@@ -106,6 +106,7 @@ require (
 	github.com/hashicorp/go-secure-stdlib/nonceutil v0.1.0
 	github.com/hashicorp/go-secure-stdlib/parseutil v0.1.8
 	github.com/hashicorp/go-secure-stdlib/password v0.1.1
+	github.com/hashicorp/go-secure-stdlib/permitpool v0.0.0-20250109173936-1ecdd6ad783f
```

Comment:
Leaving this PR as draft until this can be updated following the merge of https://github.com/hashicorp/go-secure-stdlib/pull/153